app/lib/common/widget/custom_chat_input.dart

```diff
@@ -7,10 +7,20 @@ import 'package:get/get.dart';
 import 'package:themed/themed.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 
+// ignore: must_be_immutable
 class CustomChatInput extends StatelessWidget {
-  CustomChatInput({Key? key, required this.context}) : super(key: key);
+  CustomChatInput({
+    Key? key,
+    required this.context,
+    required this.isChatScreen,
+    required this.onButtonPressed,
+    required this.roomName,
+  }) : super(key: key);
   final BuildContext context;
+  final Function()? onButtonPressed;
```

Comment:
If this is the required field, `?` may be not necessary.

Reply:
Agree.